src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs

```diff
@@ -22,6 +23,7 @@ public sealed partial class ProcessStartInfo
         private string _directory;
         private string _userName;
         private string _verb;
+        private Collection<string> _argumentList = new Collection<string>();
```

Comment:
Can this be lazily initialized only when needed?  Otherwise all existing code that creates a ProcessStartInfo is going to end up allocating a `Collection<string>`.  You'd do the lazy initialization in the ArgumentList getter, and then code like that earlier which checks to see what's in the collection would refer to the field directly rather than to the property.

Reply:
is this okay ?
https://github.com/dotnet/corefx/pull/27191/files#diff-9d3ee3c373f50d96c65ac64d5d8e8d93R66
or should I use lazy<T> wrapper ?